portfolio/src/main/webapp/script.js

```diff
@@ -79,4 +79,19 @@ function fetchBlobstoreUrlAndShowForm() {
         messageForm.action = imageUploadUrl;
         messageForm.classList.remove('hidden');
       });
-}
\ No newline at end of file
+}
+
+function showPostForm() {
```

Comment:
I think the reason that you still see the blogcontent is because there's nothing saying that it should be hidden when loading the webpage

Reply:
in the CSS the display for this container is set to none. When the page initially loads, I don't see anything until i click the button, which should set the display to block (viewable) only if the user is logged in, but currently it changes it regardless

Reply:
Then, is the problem that isLoggedIn just always returns true?

Reply:
Can we try to print out the "isLoggedIn" variable and see its value?

Reply:
I suspect this will print out just a text here, not necessarily the bool you want.